### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by telanet**
# Implementation of:
When the user clicks the button to check project details, the popup with details about the project appears.
When the user clicks the close (X) button, the popup disappears.
implement popups for both mobile and desktop screen sizes:

### CODE REVIEW

File: scripts/modal.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      <div class="popupcontheader">
+        <h2 class="desktophide">${projectList[projectList.length - 1].title}</h2>
+        <h2 class="mobilehide">Keeping track of hundreds  of components website</h2>
+        <button id="closeModal" class="closeModal">${projectList[projectList.length - 1].closebtn}</button>
+      </div>
+      <div class="popupcontlangs">
+        <ul>
+          <li>${projectList[projectList.length - 1].html}</li>
+          <li>${projectList[projectList.length - 1].bootstrap}</li>
+          <li>${projectList[projectList.length - 1].ruby}</li>
```
Comment (MrBrN197) [inline scripts/modal.js:91]: - [ ] Hello please make sure that a user is able to close the popup when they click the close button. currently clicking the close button does nothing.

image

Comment (MrBrN197) [inline scripts/modal.js:91]: - [ ] Please make sure that your popup design is responsive in both desktop and mobile version 
- [ ] look at the buttons that break out of the popup container
- [ ] make sure that the popup is not hidden by the header.
- [ ] make sure that the design stays responsive in the mobile version as well.
image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @telanet & @Blantraders  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/modal.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
```
Comment (t-yanick) [inline scripts/modal.js:91]: - [ ] As requested by the previous code reviewer, which is a requirement for this project, and best  practice as a front end developer to stick to the design as much as possible, please make sure the your `See live` and `See source` buttons stays within the popup. Please screenshot below and kindly fix that.

***The design***
telanetDesign

***Your implementation***
telaetImplementation

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @telanet, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @telanet, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**